src/components/AstronautsPage/components/AstronautsList/AstronautsList.tsx

```diff
@@ -1,4 +1,27 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import { Grid } from '@mui/material';
+import { getAstronauts } from '../../selectors/selectors';
+import type { Astronauts } from '../../types/Astronauts';
 import { AstronautsItem } from './components/AstronautsItem';
 
-export const AstronautsList = () => <AstronautsItem />;
+export const AstronautsList = () => {
+  const astronauts: Astronauts[] = useSelector(getAstronauts);
```

Comment:
type for useSelector should be provided

Reply:
changed with already typed hook useAppSelector

Reply:
fixed in the next comment